tools/logger/convert.c

```diff
@@ -915,8 +921,13 @@ static int logger_read(void)
 			/* When the address is not correct, move forward by one DWORD (not
 			 * entire struct dma_log)
 			 */
-			fseek(global_config->in_fd, -(sizeof(dma_log) - sizeof(uint32_t)),
+			ret = fseek(global_config->in_fd, -(sizeof(dma_log) - sizeof(uint32_t)),
 			      SEEK_CUR);
+			if (ret) {
+				log_err("fetch_entry() failed on seek, aborting\n");
+				ret = -errno;
```

Comment:
ret already assigned as -1

Reply:
i want return error code set by fseek in errno variable.